### ISSUE TITLE
With old PR experience, activity page doesn't work

### ISSUE THREAD

**Issue by runer112** [refined-bitbucket/refined-bitbucket#369]
With the old PR experience, loading up a PR directly to the _/activity_ page instead loads an incomplete version of the overview page. This appears to be a new issue in v3.21.0.

**Comment by runer112** [comment]
After experimenting a bit more, it seems that this only happens if the PR has inline comments. When one is present, the target is unexpectedly redirected to _/diff?w=1#chg-\<filename\>_, where _\<filename\>_ is the name/path of the file with the most recent inline comment.

**Comment by runer112** [comment]
3.22.2 update: now the activity page simply doesn't work, ever. It *always* loads an incomplete version of the overview page instead.

**Comment by runer112** [comment]
Since I could no longer work around this issue by re-navigating to the activity page, I investigated a bit further. By experimenting with the options, **I've determined that a single option is the culprit**:

> Set "Ignore whitespace" ON by default in the Old PR Experience (already natively supported in the New PR experience)

This seems to make sense considering my previous finding that this would only happen if the PR had inline comments. That causes some diff contents to appear on the activity page, which would then trigger manipulations like this. I guess that this one in particular makes some assumptions/assertions about being on the diff page, thus breaking the activity page when it triggers there?
